test/alternator/test_metrics.py

```diff
@@ -138,6 +146,25 @@ def test_batch_get_item_count(test_table_s, metrics):
         test_table_s.meta.client.batch_get_item(RequestItems = {
             test_table_s.name: {'Keys': [{'p': random_string()}, {'p': random_string()}], 'ConsistentRead': True}})
 
+KB = 1024
+def test_rcu(test_table_s, metrics):
+    with check_increases_table_metric(metrics, 'scylla_column_family_cru_total', test_table_s.name, 4):
```

Comment:
If I understand correct, this 4 is just 2*2 because we multiplied all the counters by 2 so they are integers?

Reply:
yes, you are right